src/clientManager.js

```diff
@@ -3,13 +3,31 @@ var fs = require("fs");
 var Client = require("./client");
 var mkdirp = require("mkdirp");
 var Helper = require("./helper");
+var path = require("path");
+
+var USERS_PATH = path.join(Helper.HOME, "users");
```

Comment:
Yeah that would work since you're not working with its prototype;
```
function CM() {}
CM.FOO = 'bar';
CM.prototype.bar = function () { console.log(this.FOO); };
a = new CM
a.bar(); // undefined
```